requirements.txt

```diff
@@ -3,9 +3,9 @@ pytest==4.2.1
 pymongo==3.5.1
 tabulate
 #Video dependencies
-scipy
-h5py
-opencv-python
-tensorflow
-numpy
+scipy==1.2.1
+h5py==2.9.0
+opencv-python==4.0.0.21
+tensorflow==1.13.1
+numpy==1.16.2
 keras
```

Comment:
Not for keras?

Reply:
Couldn't find a version for it. You're very welcome to add a version if you know what version we're using.